src/voice_registration.js

```diff
@@ -44,7 +56,13 @@ go.app = function() {
                     if (go.utils.check_valid_phone_number(content) === false) {
                         return 'state_r02_retry_number';
                     } else {
-                        return 'state_r03_receiver';
+                        return go.utils
+                            // get or create mama contact
+                            .get_or_create_contact('+'+content, self.im)
```

Comment:
I'd prefer we remove the `+` here and move any address normalisation to the utils